spec/compiler/parser/parser_spec.cr

```diff
@@ -132,6 +132,7 @@ module Crystal
     it_parses "~ 1", Call.new(1.int32, "~")
     it_parses "1 && 2", And.new(1.int32, 2.int32)
     it_parses "1 || 2", Or.new(1.int32, 2.int32)
+    it_parses "&- 1", Call.new(1.int32, "&-")
```

Comment:
Is support for `&+ 1` left out intentionally?

Reply:
No, I just added `&-` because there is a prefix operation `-`. I will add the specs.

Reply:
Can prefix `-` or `+` overflow? If not, maybe it's not worth adding these.

Reply:
@asterite - can on 2 complement as 0 is taken from the positive side. Dunno about systems with other bases.

Reply:
Note that I am not planning on implementing prefix `&-`, but since unitary `-` is allowed, `&-` should also work. It is only used in an stdlib string operation that can be replaced by `0 &- expr`